backend/app/core/config.py

```diff
@@ -77,7 +89,17 @@ def get_database_url(self) -> str:
     )  # Added this line
 
     # Vector Store batch processing
-    VECTOR_STORE_BATCH_SIZE: int = int(os.getenv("VECTOR_STORE_BATCH_SIZE", "100"))
+    VECTOR_STORE_BATCH_SIZE: int = int(
+        os.getenv("VECTOR_STORE_BATCH_SIZE", "100")
+    )
+
+    # KB MCP Server
+    KNOWLEDGE_BASES_MCP: str = os.getenv(
+        "KNOWLEDGE_BASES_MCP", "http:localhost:8100/mcp/"
```

Comment:
https://

Reply:
Should I add https:// while the default is to localhost?
In the `.env` the URL will have https://